2-api/limits.md

```diff
@@ -30,7 +30,7 @@ You may encounter the following error:
 $ Comment text too long.
 ```
 
-The maximum text size per request is 3000 bytes. One character does not
+The maximum text size per request is 20 KB. One character does not
 necessarily equal one byte, as different characters have different encodings.
 For example, a UTF-8 Unicode character has an encoding between 1 byte and 4
 bytes.
```

Comment:
While we're at it: Can we remove the last sentence in this paragraph (I think it's unnecessary), and add something like: "Note that the models are trained on online comments, so performance will be best on text around that length." 

Thanks!

Reply:
Done! Merging now.